docs/CONFIG.md

```diff
@@ -0,0 +1,2003 @@
+
+### MaxProfileSize
+```toml
+MaxProfileSize = '100mb' # Default
+```
+The maximum amount of disk space that profiles may consume before profiling is disabled.
+
+### CPUProfileRate
+```toml
+CPUProfileRate = 1 # Default
+```
+See https://pkg.go.dev/runtime#SetCPUProfileRate.
+
+### MemProfileRate
+```toml
+MemProfileRate = 1 # Default
+```
+See https://pkg.go.dev/runtime#pkg-variables.
+
+### BlockProfileRate
+```toml
+BlockProfileRate = 1 # Default
+```
+See See https://pkg.go.dev/runtime#SetBlockProfileRate.
+
+### MutexProfileFraction
+```toml
+MutexProfileFraction = 1 # Default
+```
+See https://pkg.go.dev/runtime#SetMutexProfileFraction.
+
+### MemThreshold
+```toml
+MemThreshold = '4gb' # Default
+```
+The maximum amount of memory the node can actively consume before profiling begins.
+
+### GoroutineThreshold
+```toml
+GoroutineThreshold = 5000 # Default
+```
+The maximum number of actively-running goroutines the node can spawn before profiling begins.
+
+## Sentry
+```toml
+[Sentry]
+Debug = false # Default
+DSN = 'sentry-dsn'
+Environment = 'dev'
+Release = 'v1.2.3'
+```
+
+
+### Debug
+```toml
+Debug = false # Default
+```
+
+
+### DSN
+```toml
+DSN = 'sentry-dsn'
+```
+
+
+### Environment
+```toml
+Environment = 'dev'
+```
+
+
+### Release
+```toml
+Release = 'v1.2.3'
+```
+
+
+## EVM
+```toml
+[[EVM]]
+ChainID = '1'
+Enabled = true # Default
+BalanceMonitorEnabled = true
+BlockBackfillDepth = 100
+BlockBackfillSkip = true
+BlockEmissionIdleWarningThreshold = '1h'
+ChainType = 'Optimism'
+EIP1559DynamicFees = false # Default
+FinalityDepth = 42
+FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
+GasBumpPercent = 10
+GasBumpThreshold = 15
+GasBumpTxDepth = 6
+GasBumpWei = '100'
+GasEstimatorMode = 'BlockHistory' # Default
+GasFeeCapDefault = '9223372036854775807'
+GasLimitDefault = '12'
+GasLimitMultiplier = '1.234'
+GasLimitTransfer = '100'
+GasPriceDefault = '9223372036854775807'
+GasTipCapDefault = '2'
+GasTipCapMinimum = '1'
+HeadTrackerHistoryDepth = 15
+HeadTrackerMaxBufferSize = 17
+HeadTrackerSamplingInterval = '1h'
+LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
+LogBackfillBatchSize = 17
+LogPollInterval = '1m'
+MaxGasPriceWei = '281474976710655'
+MaxInFlightTransactions = 19
+MaxQueuedTransactions = 99
+MinGasPriceWei = '13'
+MinIncomingConfirmations = 13
+MinimumContractPayment = '9223372036854775807'
+NonceAutoSync = false # Default
+OCRContractConfirmations = 11
+OCRContractTransmitterTransmitTimeout = '1m'
+OCRDatabaseTimeout = '1s'
+OCRObservationGracePeriod = '1s'
+OCRObservationTimeout = '1m'
+OCR2ContractConfirmations = 7
+OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
+RPCDefaultBatchSize = 17
+TxReaperInterval = '1m'
+TxReaperThreshold = '1m'
+TxResendAfterThreshold = '1h'
+UseForwarders = false # Default
+```
+
```

Comment:
Maybe we could do something like this for the chain defaults :thinking: 

EVM defaults depend on `ChainID`:

<details>
<summary>1) Ethereum</summary>
<pre><code>[[EVM]]
ChainID = '1'
Enabled = true # Default
BalanceMonitorEnabled = true
BlockBackfillDepth = 100
BlockBackfillSkip = true
BlockEmissionIdleWarningThreshold = '1h'
ChainType = 'Optimism'
EIP1559DynamicFees = false # Default
FinalityDepth = 42
FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
GasBumpPercent = 10
GasBumpThreshold = 15
GasBumpTxDepth = 6
GasBumpWei = '100'
GasEstimatorMode = 'BlockHistory' # Default
GasFeeCapDefault = '9223372036854775807'
GasLimitDefault = '12'
GasLimitMultiplier = '1.234'
GasLimitTransfer = '100'
GasPriceDefault = '9223372036854775807'
GasTipCapDefault = '2'
GasTipCapMinimum = '1'
HeadTrackerHistoryDepth = 15
HeadTrackerMaxBufferSize = 17
HeadTrackerSamplingInterval = '1h'
LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
LogBackfillBatchSize = 17
LogPollInterval = '1m'
MaxGasPriceWei = '281474976710655'
MaxInFlightTransactions = 19
MaxQueuedTransactions = 99
MinGasPriceWei = '13'
MinIncomingConfirmations = 13
MinimumContractPayment = '9223372036854775807'
NonceAutoSync = false # Default
OCRContractConfirmations = 11
OCRContractTransmitterTransmitTimeout = '1m'
OCRDatabaseTimeout = '1s'
OCRObservationGracePeriod = '1s'
OCRObservationTimeout = '1m'
OCR2ContractConfirmations = 7
OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
RPCDefaultBatchSize = 17
TxReaperInterval = '1m'
TxReaperThreshold = '1m'
TxResendAfterThreshold = '1h'
UseForwarders = false # Default
</code></pre>
</details>


<details>
<summary>3) Ropsten</summary>
<pre><code>[[EVM]]
ChainID = '1'
Enabled = true # Default
BalanceMonitorEnabled = true
BlockBackfillDepth = 100
BlockBackfillSkip = true
BlockEmissionIdleWarningThreshold = '1h'
ChainType = 'Optimism'
EIP1559DynamicFees = false # Default
FinalityDepth = 42
FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
GasBumpPercent = 10
GasBumpThreshold = 15
GasBumpTxDepth = 6
GasBumpWei = '100'
GasEstimatorMode = 'BlockHistory' # Default
GasFeeCapDefault = '9223372036854775807'
GasLimitDefault = '12'
GasLimitMultiplier = '1.234'
GasLimitTransfer = '100'
GasPriceDefault = '9223372036854775807'
GasTipCapDefault = '2'
GasTipCapMinimum = '1'
HeadTrackerHistoryDepth = 15
HeadTrackerMaxBufferSize = 17
HeadTrackerSamplingInterval = '1h'
LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
LogBackfillBatchSize = 17
LogPollInterval = '1m'
MaxGasPriceWei = '281474976710655'
MaxInFlightTransactions = 19
MaxQueuedTransactions = 99
MinGasPriceWei = '13'
MinIncomingConfirmations = 13
MinimumContractPayment = '9223372036854775807'
NonceAutoSync = false # Default
OCRContractConfirmations = 11
OCRContractTransmitterTransmitTimeout = '1m'
OCRDatabaseTimeout = '1s'
OCRObservationGracePeriod = '1s'
OCRObservationTimeout = '1m'
OCR2ContractConfirmations = 7
OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
RPCDefaultBatchSize = 17
TxReaperInterval = '1m'
TxReaperThreshold = '1m'
TxResendAfterThreshold = '1h'
UseForwarders = false # Default
</code></pre>
</details>

<details>
<summary>42) Kovan</summary>
<pre><code>[[EVM]]
ChainID = '1'
Enabled = true # Default
BalanceMonitorEnabled = true
BlockBackfillDepth = 100
BlockBackfillSkip = true
BlockEmissionIdleWarningThreshold = '1h'
ChainType = 'Optimism'
EIP1559DynamicFees = false # Default
FinalityDepth = 42
FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
GasBumpPercent = 10
GasBumpThreshold = 15
GasBumpTxDepth = 6
GasBumpWei = '100'
GasEstimatorMode = 'BlockHistory' # Default
GasFeeCapDefault = '9223372036854775807'
GasLimitDefault = '12'
GasLimitMultiplier = '1.234'
GasLimitTransfer = '100'
GasPriceDefault = '9223372036854775807'
GasTipCapDefault = '2'
GasTipCapMinimum = '1'
HeadTrackerHistoryDepth = 15
HeadTrackerMaxBufferSize = 17
HeadTrackerSamplingInterval = '1h'
LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
LogBackfillBatchSize = 17
LogPollInterval = '1m'
MaxGasPriceWei = '281474976710655'
MaxInFlightTransactions = 19
MaxQueuedTransactions = 99
MinGasPriceWei = '13'
MinIncomingConfirmations = 13
MinimumContractPayment = '9223372036854775807'
NonceAutoSync = false # Default
OCRContractConfirmations = 11
OCRContractTransmitterTransmitTimeout = '1m'
OCRDatabaseTimeout = '1s'
OCRObservationGracePeriod = '1s'
OCRObservationTimeout = '1m'
OCR2ContractConfirmations = 7
OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
RPCDefaultBatchSize = 17
TxReaperInterval = '1m'
TxReaperThreshold = '1m'
TxResendAfterThreshold = '1h'
UseForwarders = false # Default
</code></pre>
</details>

<details>
<summary>10) Optimism</summary>
<pre><code>[[EVM]]
ChainID = '1'
Enabled = true # Default
BalanceMonitorEnabled = true
BlockBackfillDepth = 100
BlockBackfillSkip = true
BlockEmissionIdleWarningThreshold = '1h'
ChainType = 'Optimism'
EIP1559DynamicFees = false # Default
FinalityDepth = 42
FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
GasBumpPercent = 10
GasBumpThreshold = 15
GasBumpTxDepth = 6
GasBumpWei = '100'
GasEstimatorMode = 'BlockHistory' # Default
GasFeeCapDefault = '9223372036854775807'
GasLimitDefault = '12'
GasLimitMultiplier = '1.234'
GasLimitTransfer = '100'
GasPriceDefault = '9223372036854775807'
GasTipCapDefault = '2'
GasTipCapMinimum = '1'
HeadTrackerHistoryDepth = 15
HeadTrackerMaxBufferSize = 17
HeadTrackerSamplingInterval = '1h'
LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
LogBackfillBatchSize = 17
LogPollInterval = '1m'
MaxGasPriceWei = '281474976710655'
MaxInFlightTransactions = 19
MaxQueuedTransactions = 99
MinGasPriceWei = '13'
MinIncomingConfirmations = 13
MinimumContractPayment = '9223372036854775807'
NonceAutoSync = false # Default
OCRContractConfirmations = 11
OCRContractTransmitterTransmitTimeout = '1m'
OCRDatabaseTimeout = '1s'
OCRObservationGracePeriod = '1s'
OCRObservationTimeout = '1m'
OCR2ContractConfirmations = 7
OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
RPCDefaultBatchSize = 17
TxReaperInterval = '1m'
TxReaperThreshold = '1m'
TxResendAfterThreshold = '1h'
UseForwarders = false # Default
</code></pre>
</details>


<details>
<summary>137) Polygon</summary>
<pre><code>[[EVM]]
ChainID = '1'
Enabled = true # Default
BalanceMonitorEnabled = true
BlockBackfillDepth = 100
BlockBackfillSkip = true
BlockEmissionIdleWarningThreshold = '1h'
ChainType = 'Optimism'
EIP1559DynamicFees = false # Default
FinalityDepth = 42
FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
GasBumpPercent = 10
GasBumpThreshold = 15
GasBumpTxDepth = 6
GasBumpWei = '100'
GasEstimatorMode = 'BlockHistory' # Default
GasFeeCapDefault = '9223372036854775807'
GasLimitDefault = '12'
GasLimitMultiplier = '1.234'
GasLimitTransfer = '100'
GasPriceDefault = '9223372036854775807'
GasTipCapDefault = '2'
GasTipCapMinimum = '1'
HeadTrackerHistoryDepth = 15
HeadTrackerMaxBufferSize = 17
HeadTrackerSamplingInterval = '1h'
LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
LogBackfillBatchSize = 17
LogPollInterval = '1m'
MaxGasPriceWei = '281474976710655'
MaxInFlightTransactions = 19
MaxQueuedTransactions = 99
MinGasPriceWei = '13'
MinIncomingConfirmations = 13
MinimumContractPayment = '9223372036854775807'
NonceAutoSync = false # Default
OCRContractConfirmations = 11
OCRContractTransmitterTransmitTimeout = '1m'
OCRDatabaseTimeout = '1s'
OCRObservationGracePeriod = '1s'
OCRObservationTimeout = '1m'
OCR2ContractConfirmations = 7
OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
RPCDefaultBatchSize = 17
TxReaperInterval = '1m'
TxReaperThreshold = '1m'
TxResendAfterThreshold = '1h'
UseForwarders = false # Default
</code></pre>
</details>

<details>
<summary>588) Metis</summary>
<pre><code>[[EVM]]
ChainID = '1'
Enabled = true # Default
BalanceMonitorEnabled = true
BlockBackfillDepth = 100
BlockBackfillSkip = true
BlockEmissionIdleWarningThreshold = '1h'
ChainType = 'Optimism'
EIP1559DynamicFees = false # Default
FinalityDepth = 42
FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
GasBumpPercent = 10
GasBumpThreshold = 15
GasBumpTxDepth = 6
GasBumpWei = '100'
GasEstimatorMode = 'BlockHistory' # Default
GasFeeCapDefault = '9223372036854775807'
GasLimitDefault = '12'
GasLimitMultiplier = '1.234'
GasLimitTransfer = '100'
GasPriceDefault = '9223372036854775807'
GasTipCapDefault = '2'
GasTipCapMinimum = '1'
HeadTrackerHistoryDepth = 15
HeadTrackerMaxBufferSize = 17
HeadTrackerSamplingInterval = '1h'
LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
LogBackfillBatchSize = 17
LogPollInterval = '1m'
MaxGasPriceWei = '281474976710655'
MaxInFlightTransactions = 19
MaxQueuedTransactions = 99
MinGasPriceWei = '13'
MinIncomingConfirmations = 13
MinimumContractPayment = '9223372036854775807'
NonceAutoSync = false # Default
OCRContractConfirmations = 11
OCRContractTransmitterTransmitTimeout = '1m'
OCRDatabaseTimeout = '1s'
OCRObservationGracePeriod = '1s'
OCRObservationTimeout = '1m'
OCR2ContractConfirmations = 7
OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
RPCDefaultBatchSize = 17
TxReaperInterval = '1m'
TxReaperThreshold = '1m'
TxResendAfterThreshold = '1h'
UseForwarders = false # Default
</code></pre>
</details>